components/supervisor/pkg/supervisor/tasks.go

```diff
@@ -377,9 +378,14 @@ func (tm *tasksManager) watch(task *task, terminal *terminal.Term) {
 	go func() {
 		defer stdout.Close()
 
-		fileName := tm.prebuildLogFileName(task)
-		// TODO(janx): If the file already exists (from a parent prebuild), extract its "time saved", and log that below
-		// (instead, or in addition to, the incremental prebuild time).
+		var (
+			fileName    = tm.prebuildLogFileName(task)
+			oldFileName = fileName + "-old"
+		)
+		if _, err := os.Stat(fileName); err == nil {
+			// If the file already exists (from a parent prebuild), temporarily move it so that it doesn't get truncated.
+			_ = os.Rename(fileName, oldFileName)
```

Comment:
What happens if the rename fails? Considering the error is ignored here, a comment as to why ignoring it is fine would be handy